src/script/api/script_company.hpp

```diff
@@ -129,8 +129,8 @@ class ScriptCompany : public ScriptObject {
 	/**
 	 * Check if a CompanyID is your CompanyID, to ease up checks.
 	 * @param company The company index to check.
+	 * @game @pre ScriptCompanyMode::IsValid().
```

Comment:
If it's a precondition, then use `EnforceCompanyModeValid(false);` in the actual implementation.

Reply:
done